File: VB/16_Print/SplitPage.vb
Imports Spire.Pdf

Namespace SplitPage
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new PdfDocument object.
			Dim doc As New PdfDocument()

			' Load the PDF file from the specified path.
			doc.LoadFromFile("..\..\..\..\..\..\Data\SplitPage.pdf")

			' Set the printing orientation to portrait (not landscape).
			doc.PrintSettings.Landscape = False

			' Select a page range with only page 1 for printing.
			doc.PrintSettings.SelectPageRange(1, 1)

			' Select the split page layout for printing.
			doc.PrintSettings.SelectSplitPageLayout()

			' Print the document.
			doc.Print()

			' Close the document after printing.
			doc.Close()
		End Sub
	End Class
End Namespace

File: CS/16_Print/SplitPage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Pdf;
using Spire.Pdf.Graphics;

namespace SplitPage
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a pdf document
            PdfDocument doc = new PdfDocument();

            //Load a file
            doc.LoadFromFile(@"..\..\..\..\..\..\Data\SplitPage.pdf");

            //Indicating whether the page is printed in landscape or portrait orientation.
            doc.PrintSettings.Landscape = false;

            //Set print page range
            doc.PrintSettings.SelectPageRange(1, 1);

            //Select split page to multiple paper layout
            doc.PrintSettings.SelectSplitPageLayout();

            //Print document
            doc.Print();

            //Close the document
            doc.Close();
        }
    }
}
